### PR TITLE
Mudit D04 homework

### PR DESCRIPTION
**Pull request by muditk**
Wasn't able to solve the Caesar cypher question. I shall need your help for it.

### CODE REVIEW

File: D04_ex00.py [hunk 1]
```diff
@@ -9,16 +9,31 @@
...
-
+from random import randint
 
 # Body
-
-
-
-
-################################################################################
+anything = randint(1, 26)
```
Comment (danielsgriffin) [inline D04_ex00.py]: I may have misled you. randint is, strangely, inclusive.

File: D04_ex00.py [hunk 2]
```diff
@@ -9,16 +9,31 @@
...
+        n = int(input("Guess a number between 1 and 25 "))
+        if n > 25:
+            print("Number out of range")
+        elif n == anything:
+            print("You guessed correctly!")
+        elif n < anything:
+            print("Too low")
+        elif n > anything:
+            print("Too high")
+        continue
```
Comment (danielsgriffin) [inline D04_ex00.py]: Is this continue necessary?

File: HW04_ch07_ex02.py [hunk 3]
```diff
@@ -21,6 +21,17 @@
 
 ###############################################################################
 # Imports
+import math
+
+def eval_loop():
```
Comment (danielsgriffin) [inline HW04_ch07_ex02.py]: "continue until the user enters 'done', and then return the value of the last expression it evaluated"

File: HW04_ch08_ex00.py [hunk 4]
```diff
@@ -21,14 +21,23 @@
 
 # Body
 
+def count(word, alphabet):
+    count = 0
+    for letter in word:
+        if letter == alphabet:
+            count = count + 1
```
Comment (danielsgriffin) [inline HW04_ch08_ex00.py]: count += 1

File: HW04_ch08_ex04.py [hunk 5]
```diff
@@ -34,15 +37,15 @@ def any_lowercase2(s):
 
 
 def any_lowercase3(s):
-    """Explain what is wrong, if anything, here.
+    """Flag returns False for test string "SupporT". Script does not work.
```
Comment (danielsgriffin) [inline HW04_ch08_ex04.py]: Why does it return False?

**Comment by danielsgriffin** [comment]
Good job. Let's work on Caesar together sometime this week.

File: HW04_ch07_ex02.py [hunk 6]
```diff
@@ -21,6 +21,17 @@
...
+    while True:
+            user_input = str(input("Enter an expression: "))
```
Comment (sandeeppal1991) [inline HW04_ch07_ex02.py]: Its a good practice to mention program use instructions when the program is run. Eg: You could mention " type 'done' when you want to finish the code"

**Pull request closed without merging**